x-pack/metricbeat/module/googlecloud/stackdriver/integration.go

```diff
@@ -35,10 +35,10 @@ func GetConfigForTest(t *testing.T, metricSetName string) map[string]interface{}
 		}
 
 		if metricSetName == "stackdriver" {
-			config["stackdriver.service"] = "compute"
 			stackDriverConfig := stackDriverConfig{
+				ServiceName: "compute",
```

Comment:
Nit. Use `googlecloud.ServiceCompute`?
```suggestion
				ServiceName: googlecloud.ServiceCompute,
```

Reply:
Sorry why we need to change to `googlecloud.ServiceCompute`? I think this function is only for constructing config for testing purpose 😄 and the ServiceName here is equivalent to the `service` under `metrics`:
```
- module: googlecloud
  metricsets:
    - stackdriver
  project_id: "your project id"
  credentials_file_path: "your JSON credentials file path"
  exclude_labels: false
  period: 1m
  metrics:
    - aligner: ALIGN_NONE
      service: compute
      metric_types:
        - "instance/cpu/reserved_cores"
        - "instance/cpu/usage_time"
        - "instance/cpu/utilization"
        - "instance/uptime"
```

Reply:
I meant to use an existing constant, but you are right, we are creating a config for a test, no need to use a constant.